stories/notifications.stories.js

```diff
@@ -11,24 +11,35 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import Notification from '../assets/js/components/notifications/notification';
-import ModulesList from '../assets/js/components/modules-list';
+import ModulesList from '../assets/js/components/ModulesList';
+import { STORE_NAME as CORE_MODULES } from '../assets/js/googlesitekit/modules/datastore/constants';
+import FIXTURES from '../assets/js/googlesitekit/modules/datastore/fixtures';
+import { WithTestRegistry } from '../tests/js/utils';
 
 global._googlesitekitLegacyData.canAdsRun = true;
 
 storiesOf( 'Global/Notifications', module )
-	.add( 'Module Setup Complete', () => (
-		<Notification
-			id="notification-id"
-			title={ __( 'Congrats on completing the setup for Analytics!', 'google-site-kit' ) }
-			handleDismiss={ () => {} }
-			winImage={ `${ global._googlesitekitLegacyData.admin.assetsRoot }images/rocket.png` }
-			dismiss={ __( 'OK, Got it!', 'google-site-kit' ) }
-			format="large"
-			type="win-success"
-		>
-			<ModulesList />
-		</Notification>
-	) )
+	.add( 'Module Setup Complete', () => {
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( CORE_MODULES ).receiveGetModules( FIXTURES );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry } >
+				<Notification
+					id="notification-id"
+					title={ __( 'Congrats on completing the setup for Analytics!', 'google-site-kit' ) }
+					handleDismiss={ () => {} }
+					winImage={ `${ global._googlesitekitLegacyData.admin.assetsRoot }images/rocket.png` }
```

Comment:
Please update the URL, using an imported image to be consistent with other stories as this recently changed.

Reply:
It seems that image imports can't be done like 
```js
winImage={ global._googlesitekitLegacyData.admin.assetsRoot + rocketImage }
```

as it results in a src 
```
http://localhost:9001/assets/static/media/rocket.3e64a093.png
```
which gives a 404.

Am I doing something wrong here? The other image files are referenced directly by the file name in the same `Notifications` story scope.

Reply:
Oh sorry, my mistake. This changed recently in the PR for #2043 and it just hasn't been merged yet but I thought it had. You're right, this should be as you have it now 👍